client/blocks/app-banner/style.scss

```diff
@@ -56,7 +56,11 @@
 	margin-top: 8px;
 }
 
-a.app-banner__no-thanks-button {
+.button.app-banner__no-thanks-button {
+
+	background: none;
+	border: 0;
```

Comment:
It would be so nice if `<Button/>` had direct support for this style. `is-borderless` and `is-link` are close, but not quite there.

Reply:
Agreed, though I'm not sure if this particular style is used elsewhere – I don't know enough about Calypso's CSS to be able to even name it properly!

 The adjustment to the button tag was only included in this PR because the automated tests wouldn't allow me to proceed without it! 😅

Reply:
There are other places where this style -- "deemphasized negative action" -- is used. For example, when editing an image in Media, there is a "Delete" button in the modal's toolbar.

Could be a nice enhancement for the Muriel component library.

Reply:
Good to know! I'm not sure how to contribute to that, but I'm happy to do so!! 😃